TurkishFilter/sections/specific.txt

```diff
@@ -1445,6 +1445,7 @@ dizimex.net###gotorek
 ||fenomenol-net.cdn.ampproject.org/*/fenomenol.net/*.mp4$domain=filmkovani.com|hdfilmcehennemi2.pw
 ||720pizleme.com/jsver1.js
 filmmodu.co##.header-menu-bottom-ads
+sigortadunyasi.com.tr##.header-main-area
```

Comment:
It causes the incorrect blocking. I will change it to `||sigortadunyasi.com.tr/wp-content/uploads/*/tsb.gif`.